AutoGraph/SubscriptionResponseSerializer.swift

```diff
@@ -1,8 +1,18 @@
 import Foundation
 import JSONValueRX
 
-enum SubscriptionResponseSerializerError: Error {
-    case failedToConvertTextToData
+public enum ResponseSerializerError: Error {
+    case failedToConvertTextToData(String)
+    case webSocketError(String)
+    
+    public var localizedDescription: String {
+        switch self {
+        case let .failedToConvertTextToData(text):
+            return "Failed to convert into data:\n\(text)"
+        case let .webSocketError(error):
+            return "Received websocket error event:\n\(error)"
```

Comment:
it's generally better to use space instead of newlines in error logs because some loggers might split it into separate log lines, e.g. we've seen this with datadog.